pkg/operation/botanist/component/istio/type.go

```diff
@@ -16,4 +16,6 @@ package istio
 
 const (
 	istioReleaseName = "istio"
+	// GatewayPort is the port exposed by the istio ingress gateway
+	GatewayPort = 8132
```

Comment:
I don't quite get why you have moved this now. Didn't you just argue in https://github.com/gardener/gardener/pull/4242#discussion_r656879036 that the port is related to `vpnseedserver`? My comment was only about improving the doc string to make it more clear.

Anyways, I'll approve the PR now.